src/integration/integrationTestEnvironment.js

```diff
@@ -18,7 +18,21 @@ class IntegrationTestEnvironment extends JsdomEnvironment {
     } = context.docblockPragmas;
     const pageType = getPageTypeFromTestPath(context.testPath);
 
-    const platformForPath = platform === 'canonical' ? '' : `.${platform}`;
+    let platformForPath = '';
+
+    switch (platform) {
+      case 'canonical':
+        platformForPath = '';
+        break;
+      case 'amp':
+        platformForPath = '.amp';
+        break;
+      case 'lite':
+        platformForPath = '.lite';
+        break;
+      default:
+        platformForPath = '';
+    }
```

Comment:
This could be simplified even further (as the `canonical` & `default` cases are identical):

either

```suggestion
    let platformForPath = '';

    switch (platform) {
      case 'amp':
        platformForPath = '.amp';
        break;
      case 'lite':
        platformForPath = '.lite';
        break;
      default:
        platformForPath = '';
    }
```

OR 
```suggestion
    const platformForPath = ['amp', 'lite'].includes(platform) ? `.${platform}` : '';
```

Reply:
Oh yh both of these look good. I liked the idea of using the switch statement to make it clearer what the logic is. But I think the one liner is still better and clearer than what we had before.